Agile_Essay.tex

```diff
@@ -0,0 +1,97 @@
+
+\section{Introduction}
+
+Agile processes are widely used in the Game Development industry, they encourage a focus on self-organising teams along with regular reflection, and communication \cite{Agile}. Due to the scope involved in game development tasks, working within teams, rather than individually, is necessary. However, such focus on teamwork and communication can be problematic for introverts. A correlation has been found between an individual's personality type, based on the Five-Factor model (FFM), and their preference for agile processes \textit{`The data supports a positive relationship between extraversion and agile preference as well as openness and agile preference'} \cite{BishopDeokar}. FFM marks personality traits across a continuum of five distinct strands, namely: Extraversion, Openness, Agreeableness, Neuroticism, and Conscientiousness. Although individuals are rarely able to be placed in one category exclusively, there is usually a tendency towards certain factors in FFM, and therefore, this tool can still be useful in managing effects of personality on the workplace.
+
+Subjects with introverted or neurotic personality types were noted as less likely to be inclined towards agile. Such findings are not unique to the Agile Development process, workplaces in general tend to favour those who are extroverted and open. As Susan Cain explains, \textit{`studies show that we rank fast and frequent talkers as more competent, likeable, and even smarter than slow ones' }\cite{Cain}. Such views are also seen in educational settings \textit{`research suggests that the majority of our teachers believe that the 'ideal student' is an extrovert.'} \cite{Cain}. This paper sets out to explore how introverted personality types can best adapt to Agile and how they can be aided in doing so.
+
+
+\section{Introversion, the Games Industry and Agile}
+
+It is often assumed that introverts are well suited to computing, \textit{`People in and out of the computing field often believe that a) technology is a field that is very attractive to introverts, and b) technology is a field that has traditionally been very amenable to introverts'} \cite{Kacz}, and certainly in early hacker culture, there is evidence to support such beliefs. Yet, in computing, especially within the games industry, teamwork has become increasingly important. Today, as opposed to the height of hacker culture, programming is conducted in teams, often large and evolving, and as such programming may not be as ideal to the introverted as it once was. 

```

Comment:
Avoid paragraphs with fewer than 3 sentences. The first sentence could be restructured into two or more smaller sentences with ease. For example:
```
It is often assumed that introverts are well suited to computing. 
Kacz notes that \textit{`people in and out of the computing field often believe that 
(a) technology is a field that is very attractive to introverts, and 
(b), technology is a field that has traditionally been very amenable to introverts'} \cite{Kacz}. 
While this may have been true for early hacker culture, ...
```